yarn-project/pxe/src/storage/note_data_provider/note_data_provider.ts

```diff
@@ -251,6 +251,11 @@ export class NoteDataProvider {
   async getNotes(filter: NotesFilter): Promise<NoteDao[]> {
     filter.status = filter.status ?? NoteStatus.ACTIVE;
 
+    // throw early if scopes is an empty array
+    if (filter.scopes !== undefined && filter.scopes.length === 0) {
+      throw new Error('Trying to get notes with an empty scopes array');
```

Comment:
```suggestion
      throw new Error('Trying to get notes with an empty scopes array. Scopes have to be set to undefined if intending on not filtering by scopes.');
```
From the original error message it would not be clear how exactly to fix the error.